lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -4616,6 +4616,20 @@ private void createAnnouncementsDialog(UIContainer tofill, SimplePage currentPag
 	// in the edit dialogs
 	private void createAddMultimediaDialog(UIContainer tofill, SimplePage currentPage) {
 		UIOutput.make(tofill, "add-multimedia-dialog").decorate(new UIFreeAttributeDecorator("title", messageLocator.getMessage("simplepage.resource")));
+
+		String max = ServerConfigurationService.getString("content.upload.max", null);
```

Comment:
String max = ServerConfigurationService.getString("content.upload.max", 20);

Reply:
The it should be:

int max = ServerConfigurationService.getInt("content.upload.max", 20);

Or 

String max = ServerConfigurationService.getString("content.upload.max", "20");